runtime/doc/terminal.txt

```diff
@@ -187,26 +187,40 @@ event. The event is handled directly by Nvim and is not forwarded to plugins.
 
 OSC 133: shell integration			*terminal-osc133*
 
-Some shells will emit semantic escape sequences (OSC 133) to mark the
-beginning and end of a prompt. The start of a prompt is marked with the
-sequence `OSC 133 ; A`. Nvim can be configured to create signs in terminal
-buffers marking shell prompts. Example: >lua
+Shells can emit semantic escape sequences (OSC 133) to mark where each prompt
+starts and ends. The start of a prompt is marked by sequence `OSC 133 ; A ST`,
+and the end by `OSC 133 ; B ST`.
```

Comment:
Also mention "end" sequence, and ST in both.

Reply:
Technically `ST` isn't required, since OSCs can also be terminated by a `BEL`. In either case it's irrelevant for plugin authors, since the terminator is not included in the `TermRequest` sequence.

Reply:
I was thinking from the perspective of a user (me) trying to configure their shell. I get weird behavior if `\007` is not included in the PS1/PROMPT_COMMAND values.

Reply:
`\007` is `BEL`, `\x1b\\` (`ESC \`) is `ST`. The sequence does need a terminator, but it can be either `BEL` or `ST`. That's all I was saying.

This is probably fine as-is. It's not worth making this text more confusing by saying "BEL or ST".